### ISSUE TITLE
guifont no effect

### ISSUE THREAD

**Issue by GNUqb114514** [glacambre/firenvim#1648]
- OS Version: Arch Linux x86_64
- Browser Version: Firefox 132.0.1
- Browser Addon Version: 0.2.16
- Neovim Plugin Version: commit 64f9389

### What I tried to do

Change `guifont`:
```lua
return {
    { 'glacambre/firenvim', build = ":call firenvim#install(0)",
        lazy = false,
        config = function ()
            if vim.g.started_by_firenvim then
                vim.o.laststatus=0
                vim.opt.guifont='UbuntuMono Nerd Font Mono'
            end
        end
    },
}
```

### What happened

The `guifont` was set correctly, but the font stay unchanged:
![图片](https://github.com/user-attachments/assets/f443e097-e1c4-47f9-9a49-8bba7a3146c2)

**Comment by theherk** [comment]
Yeah, something is strange here. If I set the fonts manually and disable "Allow pages to choose their own fonts, instead of your selections above" I can set the font for instance. That isn't ideal though, because I just want to set the font of the neovim. I tried putting this in userContext.css.

```css
textarea, textarea * {
  font-family: "VictorMono Nerd Font", "Victor Mono", monospace !important;
}
```
This does correctly set the font for the textarea, but this is not picked up by the instance of Neovim. Would be really nice to control the font in here.
